### DISCUSSION TITLE
@cds vs @clr

### DISCUSSION THREAD

**Discussion by KALMOGO** [vmware/clarity#6766] [Q&A]
Sorry ! I notice that there is two clarity libaries: @cds nd @clr . I am still using @clr. So what are the differences and more important it 's possible to combine the  both in the same projet without any issues? Thank you so much!

it is because of that : https://clarity.design/storybook

**Comment by bbogdanov** [comment]
Hi @KALMOGO,

Attaching some links to a blog posts we wrote about the difference and how you can use them together. 

https://medium.com/claritydesignsystem/level-up-your-application-by-adopting-clarity-core-8a5f3f863139

https://medium.com/claritydesignsystem/clarity-core-72f6d3a029bc
